docs/developer-docs/cycles-and-transaction-costs.md

```diff
@@ -0,0 +1,234 @@
+
+:::info
+While this may be referred to as the ICP reverse 'gas' model, it is important to note that ICP does not use 'gas' -- it uses cycles.
+:::
+
+You can learn more about the ICP reverse gas model [here](https://internetcomputer.org/capabilities/reverse-gas).
+
+Each canister deployed on ICP has a cycles balance. When a canister is first deployed, it must be deployed with an amount of cycles that covers the cost of creating the canister and then provides enough cycles for the canister's operations. If a canister's cycles balance becomes depleted and it cannot cover the cost of the canister's operations, the canister must be 'topped up', which refers to depositing additional cycles into the canister's cycles balance.
+
+You can learn how to query a canister's cycles balance [here](/docs/current/tutorials/developer-journey/level-1/1.6-managing-canisters#checking-the-cycles-balance-of-a-canister), and you can learn how to top up a canister [here](/docs/current/tutorials/developer-journey/level-1/1.6-managing-canisters#topping-up-a-canisters-cycles-balance).
+
+
+### How do ICP costs compare to the EVM gas model?
+
+On other blockchain platforms, such as Ethereum, users are responsible for paying the 'gas' charge for every on-chain interaction they perform. For some dapps, that means the user must pay a gas fee for doing tasks as simple as commenting on or liking a video. ICP removes this expensive barrier for users by allowing them to interact with dapps deployed on the network without paying gas fees, instead charging the canisters themselves for the resources used.
+
+Additionally, gas fees on Ethereum can fluctuate in price unpredictably, resulting in drastically different gas fees based on the current price of ETH and the current network conditions. This can make using dapps on Ethereum difficult and expensive, as one day it may cost significantly more than the day prior, or vice versa. Since ICP uses cycles, which have a fixed price based on XDR, the price is predictable and stable.
+
+One downside of the ICP cycles model in comparison to the EVM gas model is that it requires prerequisite steps and ongoing maintenance for developers. ICP tokens must be converted into cycles, then cycles must be deposited into a canister's cycles balance. Canisters must have their cycles balances maintained and regularly topped up as they continuously use resources; otherwise, the canister will be removed from the network if it runs out of cycles. However, a [freezing threshold](/docs/current/tutorials/developer-journey/level-1/1.6-managing-canisters#setting-the-canisters-freezing-threshold) can be set that pauses a canister's executions if the cycles amount is expected to fall below a certain amount, and there are several community tools that have been developed to automate managing a canister's cycles, such as [CycleOps](https://cycleops.dev/).
+
+
+## Canister transactions
```

Comment:
Let's avoid the word "transaction" because IC doesn't have transactions that people coming from Ethereum would expect. Something like a "call" or a "message" would be better.

Reply:
Changed "Transaction" to "calls"